spec/expeditor/service_spec.rb

```diff
@@ -89,16 +89,18 @@
   end
 
   describe '#status' do
-    let(:service) { Expeditor::Service.new(sleep: 10) }
     it 'returns current status' do
+      # Set large value of period in case test takes long time.
+      service = Expeditor::Service.new(period: 100)
```

Comment:
In other test cases, Expeditor uses ordinary variable assignment not using `let`. And since this `service` is not shared by multiple examples, there are no advantage to use `let` here.